docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -765,7 +766,11 @@ def source(self, follow_additional_fields=True, document_type="pdf"):
           if hasattr(self, 'id') and self.id:
             content += "id: " + self.id + "\n"
           else:
-            content += "id: " + oneline(self.question_text) + "\n"
+            without_bad_chars = varname(oneline(self.question_text))
+            if len(without_bad_chars) == 0:
```

Comment:
Good catch. If someone only put in two spaces, it'd end up with no id, even if `varname()` was not used above.

Reply:
Fair, but there are a lot of places where we let people have invalid input (whatever invalid means).  Will probably end up doing another pass for stuff like that, similar to this PR of just focusing on escaping quotes

Reply:
Unique ID sounds like a good separate issue to put on our list